Integrations/integration-Cisco-umbrella.yml

```diff
@@ -0,0 +1,885 @@
+            }
+        };
+
+        return createTableEntry("Domain Security Info:", table, context);
+    }
+
+    var getDomainTags = function(domain) {
+        domain = domainWithoutProtocol(domain);
+        var url = baseURL + '/domains/' + encodeURIComponent(domain) + '/latest_tags';
+        var tags = sendRequest(url);
+
+        var table = tags.map(function(tag) {
+            return {
+                'Period Begin': tag.period.begin,
+                'Period End': tag.period.end,
+                Category: tag.Category,
+                Url: tag.url
+            }
+        });
+
+        var context = {
+            Name: domain,
+            Tags: tags.map(function(tag) {
+                return {
+                    Period: {
+                        Begin: tag.period.begin,
+                        End: tag.period.end
+                    },
+                    Category: tag.Category,
+                    Url: tag.url
+                };
+            })
+        };
+
+        return createTableEntry("Tags:", table, context);
+    }
+
+    var getDomainDNSHistory = function(domain) {
+        domain = domainWithoutProtocol(domain);
+        var url = baseURL + '/dnsdb/name/a/' + encodeURIComponent(domain) + '.json';
+        var res = sendRequest(url);
+
+        var features = res && res.features;
+        if (!features) {
+            throw 'Got an illegal response\n.Response: ' + JSON.stringigy(res);
+        };
+        var address = res.rrs_tf && res.rrs_tf[0] && res.rrs_tf[0].rrs && res.rrs_tf[0].rrs[0] && res.rrs_tf[0].rrs[0].rr;
+        features.ip = address;
+
+    /*    var t = {
+            features: res.features,
+            rr: res.rrs_tf[0].rrs[0].rr
+        }*/
+
+        var table = {};
+        var DNSHistoryContext = {};
+
+        // change features object with the following:
+        // - change keys to pascal cases
+        // - change array-values to string
+        // - special handle 'locations' key
+        Object.keys(features).forEach(function(key) {
+            if (key === 'locations') {
+                // handle locations
+            } else {
+
+                var tableKey = keyToTableKey(key);
+                // "OneTwo" to "OneTwo"
+                var contextKey = keyToContextKey(key);
+
+                var value = features[key];
+
+                // table
+                if (value instanceof Array) {
+                    // array-values
+                    table[tableKey] = value.join();
+                } else {
+                    table[tableKey] = value;
+                }
+                // context
+                DNSHistoryContext[contextKey] = value;
+            }
+        });
+
+        context = {
+            IP: {
+                Address: address,
+                DNSHistory: DNSHistoryContext
+            }
+        }
+        return createTableEntry("DNS History:", table, context);
+    }
+
+    var IpDnsFeatureInfo = {
```

Comment:
same here - put in declaration section or inside the using function if only used once